generator/beat/{beat}/scripts/clean_vendor.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
I think this one is the same (or almost the same) file we have in the top level beats/script directory. I wonder if we could copy this over instead?